mobile/src/main/java/org/openhab/habdroid/model/OpenHABNFCActionList.java

```diff
@@ -39,7 +39,8 @@ public OpenHABNFCActionList(OpenHABWidget openHABWidget) {
 					actionCommands.add("OFF");
 					actionNames.add("Toggle");
 					actionCommands.add("TOGGLE");
-				} else if (openHABWidget.getItem().getType().equals("RollershutterItem")) {
+				} else if ((openHABWidget.getItem().getType().equals("RollershutterItem")) ||
```

Comment:
consistently this line should be:

    "RollershutterItem".equals(openHABWidget.getItem().getType())